components/button/skin.css

```diff
@@ -10,32 +10,6 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-.spectrum-Button {
-  --spectrum-button-m-accent-fill-texticon-background-color: var(--spectrum-semantic-cta-background-color-default);
-  --spectrum-button-m-accent-fill-texticon-background-color-hover: var(--spectrum-semantic-cta-background-color-hover);
-  --spectrum-button-m-accent-fill-texticon-background-color-down: var(--spectrum-semantic-cta-background-color-down);
-  --spectrum-button-m-accent-fill-texticon-background-color-key-focus: var(--spectrum-semantic-cta-background-color-key-focus);
-  --spectrum-button-m-accent-fill-texticon-background-color-key-focus: var(--spectrum-semantic-cta-background-color-key-focus);
-
-  --spectrum-button-m-accent-outline-texticon-border-color: var(--spectrum-semantic-cta-background-color-default);
-  --spectrum-button-m-accent-outline-texticon-border-color-hover: var(--spectrum-semantic-cta-background-color-hover);
-  --spectrum-button-m-accent-outline-texticon-border-color-down: var(--spectrum-semantic-cta-background-color-down);
-  --spectrum-button-m-accent-outline-texticon-border-color-key-focus: var(--spectrum-semantic-cta-background-color-key-focus);
-  --spectrum-button-m-accent-outline-texticon-border-color-key-focus: var(--spectrum-semantic-cta-background-color-key-focus);
-
-  --spectrum-button-m-accent-outline-texticon-icon-color: var(--spectrum-semantic-cta-background-color-default);
-  --spectrum-button-m-accent-outline-texticon-icon-color-hover: var(--spectrum-semantic-cta-background-color-hover);
-  --spectrum-button-m-accent-outline-texticon-icon-color-down: var(--spectrum-semantic-cta-background-color-down);
-  --spectrum-button-m-accent-outline-texticon-icon-color-key-focus: var(--spectrum-semantic-cta-background-color-key-focus);
-  --spectrum-button-m-accent-outline-texticon-icon-color-key-focus: var(--spectrum-semantic-cta-background-color-key-focus);
-
-  --spectrum-button-m-accent-outline-texticon-text-color: var(--spectrum-semantic-cta-background-color-default);
-  --spectrum-button-m-accent-outline-texticon-text-color-hover: var(--spectrum-semantic-cta-background-color-hover);
-  --spectrum-button-m-accent-outline-texticon-text-color-down: var(--spectrum-semantic-cta-background-color-down);
-  --spectrum-button-m-accent-outline-texticon-text-color-key-focus: var(--spectrum-semantic-cta-background-color-key-focus);
-  --spectrum-button-m-accent-outline-texticon-text-color-key-focus: var(--spectrum-semantic-cta-background-color-key-focus);
-}
-
```

Comment:
Looks like these changes came along for the ride, maybe merge `main` since they're already there, but it probably doesn't matter.